src/components/ui/FileBrowser/Crumbs.tsx

```diff
@@ -37,11 +37,13 @@ export default function FileListCrumbs(): JSX.Element {
                 variant="text"
                 className="rounded-md hover:bg-primary-light/20 hover:!text-black focus:!text-black transition-colors cursor-pointer"
                 onClick={() => {
-                  if (index === 0) {
-                    fetchAndFormatFilesForDisplay(`${currentFileSharePath}`);
-                  } else {
+                  if (index === 0 && currentFileSharePath) {
                     fetchAndFormatFilesForDisplay(
-                      `${currentFileSharePath}?subpath=${dirArray.slice(1, index + 1).join('/')}`
+                      `${currentFileSharePath.name}`
+                    );
+                  } else if (currentFileSharePath) {
```

Comment:
Looks like this could be extracted to an enclosing if statement, something like this:
```
if (currentFileSharePath) {
  if ( index === 0 ){
  } 
  else {
  }
}
```